src/cities/cityConfigs.js

```diff
@@ -8,14 +8,16 @@ const configs = {
     hostname: 'raumfrei.neuburg-schrobenhausen.de',
     formsEnabled: true,
     title: 'Raumfrei Neuburg-Schrobenhausen',
-    logo: null
+    logo: 'neuburg_logo.svg',
```

Comment:
Maybe we should have full URLs here, so we'd have more flexibility with the actual server/address of the image?

Reply:
This would only make sense if the images are hosted on different servers. Will this ever happen?  
Nevertheless we can, of course, return the full URLs.

Reply:
I suggest leaving it as it is right now because it is easier to configure the base url in the frontend than in the backend. We can change it when we need it.

Reply:
It's not easier in the Frontend (when we look at the xamarin app). We might use the city config there as well for loading the appropriate images, so we might consider doing it already with the full url : thinking: